keyboards/clueboard/card/config.h

```diff
@@ -15,10 +15,7 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-#ifndef CONFIG_H
-#define CONFIG_H
-
-#include "config_common.h"
```

Comment:
This is still needed. 

```suggestion
#include "config_common.h"
```